site/components/IconsList.tsx

```diff
@@ -9,13 +11,34 @@ interface Props {
 }
 
 const IconsList: React.FC<Props> = ({ icons }) => {
+  const [text, setText] = useState<string>("");
+
+  const handleChangeText = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setText(e.target.value.trim());
+  };
+
+  const renderIcons = () => {
+    const searchedIcons = icons.filter((icon) =>
+      icon.name.toLowerCase().includes(text.toLowerCase())
+    );
+
+    return (
+      <>
+        <Grid item md={12} sm={12} xs={12}>
+          {searchedIcons.length} matching results
```

Comment:
Thanks for implementing this feature, It would be nicer to display "X matching results" when users search for icons.